examples/twittermap/web/public/javascripts/common/services.js

```diff
@@ -122,6 +124,46 @@ angular.module('cloudberry.common', ['cloudberry.mapresultcache'])
       }
       return filter;
     }
+    
+    // Request all data required by timeseriescache.
```

Comment:
Why do we need to have a new TimeSeries Request query?

Reply:
By far, byTimeRequest or byGeoRequest only give the result of timeseries ({day, count}) or map result ({county, count, population}). If we want to use this cache to generate both, we can have {county, day, count, population} by sending this one request. Also, no new request needs to be sent when a time interval is selected and we can use it to display timeseries histogram for each selected level (e.x.county) region.